packages/api-reference/src/types.ts

```diff
@@ -7,13 +7,29 @@ import type {
 import type { OpenAPIV2, OpenAPIV3, OpenAPIV3_1 } from '@scalar/openapi-parser'
 import type { ThemeId } from '@scalar/themes'
 import type { UseSeoMetaInput } from '@unhead/schema'
-import type { HarRequest } from 'httpsnippet-lite'
-import type {
-  ClientInfo,
-  TargetInfo,
-} from 'httpsnippet-lite/dist/types/targets/targets'
+import type { HarRequest, TargetId } from 'httpsnippet-lite'
 import type { Slot } from 'vue'
 
+// ---------------------------------------------------------------------------
+// Types copied from package as they are not exported
+type ClientInfo = {
+  key: string
+  title: string
+  link: string
+  description: string
+}
+export type TargetInfo = {
+  key: TargetId
+  title: string
+  extname: `.${string}` | null
+  default: string
+}
+
+export type AvailableTarget = TargetInfo & {
+  clients: ClientInfo[]
+}
+// ---------------------------------------------------------------------------
+
```

Comment:
The types are exported through `httpsnippet-lite/dist/types/targets/targets`, aren’t they? 👀

Reply:
Was not playing nice with the TS update (or the tsconfig change not sure exactly). But I also don't feel comfortable grabbing unexposed types deep down. Its annoying to have to manually declare them but if the original package changes then we will get compiler errors so it's at least safe.